feature-libs/product-configurator/rulebased/components/attribute/header/configurator-attribute-header.component.ts

```diff
@@ -153,17 +153,39 @@ export class ConfiguratorAttributeHeaderComponent
       .getConfiguration(this.owner)
       .pipe(take(1))
       .subscribe((configuration) => {
+        let groupId;
         if (this.groupType === Configurator.GroupType.CONFLICT_GROUP) {
-          const groupId = this.attribute.groupId;
-          if (groupId) {
-            this.configuratorGroupsService.navigateToGroup(
-              configuration,
-              groupId
-            );
-            this.focusAttribute(this.attribute.name);
-          }
+          groupId = this.attribute.groupId;
+        } else {
+          groupId = this.findConflictGroupId(configuration, this.attribute);
         }
+        if (groupId) {
+          this.configuratorGroupsService.navigateToGroup(
+            configuration,
+            groupId
+          );
+          this.focusAttribute(this.attribute.name);
+        }
+      });
+  }
+
+  findConflictGroupId(
+    configuration: Configurator.Configuration,
+    currentAttribute: Configurator.Attribute
+  ): string {
+    let groupId = '';
+    configuration.flatGroups
+      .filter(
+        (group) => group.groupType === Configurator.GroupType.CONFLICT_GROUP
+      )
+      .forEach((group) => {
```

Comment:
Shouldn't we stop after we found the first conflict group with the desired attribute? This will always loop through all conflict groups

Reply:
@Uli-Tiger can we use **find** instead?